js-exercises/alphabetic-shift/alphabeticShift.test.js

```diff
@@ -0,0 +1,20 @@
+import { alphabeticShift } from './alphabeticShift';
+
+describe('Alphabetic Shift Test Cases', () => {
+  it('returns a shifted string', () => {
+    expect(alphabeticShift('Pseudopseudohypoparathyroidism'))
+      .toEqual('Qtfvepqtfvepizqpqbsbuizspjejtn');
+
+    expect(alphabeticShift('Floccinaucinihilipilification'))
+      .toEqual('Gmpddjobvdjojijmjqjmjgjdbujpo');
+
+    expect(alphabeticShift('Floccinaucinihilipilification'))
+      .toEqual('Gmpddjobvdjojijmjqjmjgjdbujpo');
+
+    expect(alphabeticShift('Antidisestablishmentarianism'))
+      .toEqual('Boujejtftubcmjtinfoubsjbojtn');
+
+    expect(alphabeticShift('supercalifragilisticexpialidocious'))
+      .toEqual('tvqfsdbmjgsbhjmjtujdfyqjbmjepdjpvt');
+  });
+});
```

Comment:
I think if you had written an extra test, you'd have thought of few other edge cases too :)
Example, input string containing a char `z` or `Z`